analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Verifier.cs

```diff
@@ -225,7 +225,12 @@ private IEnumerable<Compilation> CompileRazor()
 
                     // Set TargetFramework
                     destinationXml.Descendants("TargetFramework").Single().Value = builder.RazorFramework;
-
+                    // Add References
+                    var itemGroup = destinationXml.Descendants("ItemGroup").Single();
+                    foreach (var reference in builder.References)
+                    {
+                        itemGroup.Add(new XElement("ProjectReference", new XAttribute("Include", reference.Display)));
```

Comment:
For my understanding: `reference.Display` points to the DLL path location?

Reply:
Yes correct